src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -634,5 +1012,72 @@ internal bool TryGetSpanOnHigherBuffer(SnapshotSpan snapshotSpan, ITextBuffer ta
             span = default;
             return false;
         }
+
+        [NonCopyable]
+        private struct State
+        {
+            /// <summary>
+            /// The current expansion session.
+            /// </summary>
+            public IVsExpansionSession _expansionSession;
+
+            /// <summary>
+            /// The set of symbols initially identified as candidates for providing arguments. When a snippet is
+            /// constructed with parameters for a specific symbol, <see cref="_symbol"/> will identify the specific
+            /// symbol from within this collection matching the current session.
+            /// </summary>
+            public ImmutableArray<IMethodSymbol> _symbols;
```

Comment:
```suggestion
            public ImmutableArray<IMethodSymbol> _methods;
```

Note: i actually strongly want this rename.  I found the name _symbols actually very vague and hard to read when doing the PR.  Making it clear that these are the contextual methods we may be showing arg help for would read a lot better IMO.

Reply:
⏱ renamed